pkg/utils/cloudconfig/azure/config.go

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+// Package azure provides utilities to load, parse, and validate Azure cloud configuration.
+package azure
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"sigs.k8s.io/cloud-provider-azure/pkg/azclient"
+	"sigs.k8s.io/cloud-provider-azure/pkg/azclient/policy/ratelimit"
+)
+
+// CloudConfig holds the configuration parsed from the --cloud-config flag
+type RateLimitConfig ratelimit.Config
+type CloudConfig struct {
+	azclient.ARMClientConfig `json:",inline" mapstructure:",squash"`
+	azclient.AzureAuthConfig `json:",inline" mapstructure:",squash"`
+	*RateLimitConfig         `json:",inline" mapstructure:",squash"`
```

Comment:
You will also need to validate and add default value for this field: https://github.com/Azure/kube-egress-gateway/blob/main/pkg/config/config.go#L90-L109

Reply:
done